tests/e2e/testcases/validatormanager/suite.go

```diff
@@ -204,22 +204,23 @@ var _ = ginkgo.Describe("[Validator Manager POA Set Up]", ginkgo.Ordered, func()
 		avaGoBootstrapValidators, err := getBootstrapValidator(uris[0])
 		gomega.Expect(err).Should(gomega.BeNil())
 		ownerAddress := common.HexToAddress(ewoqEVMAddress)
+		proxyAddress := common.HexToAddress(ProxyContractAddress)
 		subnetSDK := blockchainSDK.Subnet{
-			SubnetID:            subnetID,
-			BlockchainID:        blockchainID,
-			OwnerAddress:        &ownerAddress,
-			RPC:                 rpcURL,
-			BootstrapValidators: avaGoBootstrapValidators,
+			Network:                      network.SDKNetwork(),
+			SubnetID:                     subnetID,
+			ValidatorManagerRPC:          rpcURL,
+			ValidatorManagerBlockchainID: blockchainID,
+			ValidatorManagerAddress:      &proxyAddress,
+			ValidatorManagerOwnerAddress: &ownerAddress,
+			BootstrapValidators:          avaGoBootstrapValidators,
 		}
 
```

Comment:
can we have an E2E that tests the command to deploy validator manager controller that is not v2.0.0 (for the feature command to deploy VMC, proxy, proxy admin (supports any given VMC, not only v2.0.0)

Reply:
added (really using poa v2.0.0, but could be any contract)